src/components/ArticlePreview.jsx

```diff
@@ -35,7 +40,13 @@ export default class ArticlePreview extends React.Component {
 
           {/* Article teaser */}
           <p className="article-preview__content__teaser">{article.teaser}</p>
+          <SharingButtons
```

Comment:
Yeah I'll run it by them along with the rest of the UI closer to the release so we can tweak everything as needed